tests/jenkins/install_neuron.sh

```diff
@@ -22,6 +22,10 @@ set -e
 source ${JENKINS_DIR:-.}/_env_setup.sh
 
 patch_neuron
-spack install neuron+debug@develop
+spack install -v neuron+debug@develop
 source $SPACK_ROOT/share/spack/setup-env.sh
 module av neuron
+
+# Install reportinglib with spack to run reportinglib tests
+spack install reportinglib%intel
```

Comment:
Maybe we should rename this file to `install_neuron_reportinglib.sh` since we also now install `reportinglib` here and also rename the stage which runs this script to `Spack Installations`